packages/docs/content/components/checkbox.mdx

```diff
@@ -89,18 +89,46 @@ This component also makes use of form field styles, which can be customized by t
 - You may need to add the className `.ds-u-margin--0` to your child element(s) to avoid extra top margin
 - If you opt for smaller radio buttons or checkboxes, add className `.ds-c-choice__checkedChild--small` to your checked child container
 
-### Accessibility
+### Accessibility testing
 
-- Surround a related set of choices with a `<fieldset>`. The `<legend>` provides context for the grouping. Do not use `fieldset` and `legend` for a single checkbox.
-- Some screen readers read the `legend` text for each `fieldset`, so it should be brief and descriptive.
+#### General observations
+
+- No other interactive elements (such as links or buttons) should be included inside the label or hint text.
+- A group of checkboxes should be wrapped in a `<fieldset>` that includes a `<legend>` element within.
+  - The `<legend>` provides context for the grouping, much like a label.
+  - Do not use `fieldset` and `legend` for a single checkbox element.
 - Each input should have a semantic `id` attribute, and its corresponding `label` should have the same value in its `for` attribute.
-- The custom checkboxes and radio buttons here are accessible to screen readers because the default fields are moved off-screen.
-- `checkedChildren` will be announced to screen readers when they are exposed. They have been tested with the following devices:
```

Comment:
I wonder if some variant of this would be helpful to carry on in the new documentation. It doesn't seem to be a scenario covered in the "screen reader testing" section

Reply:
the scope for this work's changed somewhat - we're going to adopt dan and tiffany's a11y docs as-is and create follow-up tickets addressing any issues we see with...

- accessibility issues on the components themselves
- contradictions that exist across themes or different sections of the docs site

Reply:
sounds good 👍